### PR TITLE
Please review this @casey-collab

### PR DESCRIPTION
**Pull request by gonzocher**

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: english/explore-outside/help-child.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+As you approach the small, green child fishing in the marshmallow boat, you realize
```
Comment (casey-collab) [inline english/explore-outside/help-child.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: english/explore-outside/help-child.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+Do you:
+
+[return to the marshmallow room to eat more marshmallows because you no longer
+feel the need to help?](../eating-marshmallows/eating-marshmallow.md)
+
+[head now to the tower because you can see the child is safe in this one foot
+deep lake?](./journey-to-flaming-tower/journey-to-flaming-tower.md)
+
+[give the child your extra marshmallow to repair the boat?](./repair-boat/repair-boat.md)
```
Comment (casey-collab) [inline english/explore-outside/help-child.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/explore-outside/help-child.md (hunk 1 above)
Comment (casey-collab) [inline english/explore-outside/help-child.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by gonzocher** [comment]
i deleted the errant file.  I think I am good to be merged now.  Thanks.

**Comment by cbuckey-uda** [comment]
@casey-collab, please take another look

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Can you resolve the merge conflicts locally and update the pull request?

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to resolve the merge conflicts, feel free to re-open it.

**Pull request closed without merging**